worker/storageprovisioner/filesystem_ops.go

```diff
@@ -234,7 +234,7 @@ func removeFilesystems(ctx *context, ops map[names.FilesystemTag]*removeFilesyst
 			reschedule = append(reschedule, ops[tag])
 			statuses = append(statuses, params.EntityStatusArgs{
```

Comment:
Ideally we would see if the error is fatal - the detach operation is rescheduled and may succeed again second time.
One option is to wait till we get X errors and only then mark the status as error and instead of rescheduling, just leave the status as error.

Reply:
ah, sorry, ignore me. we are still scheduling the retry even setting the status to Error.